### ISSUE TITLE
IDEA: Consider a native app wrapper

### ISSUE THREAD

**Issue by WheresHJ** [codeforboston/mbta-ninja#60]
Might be good to get this in the various app stores, so perhaps a PhoneGap wrapper would be interesting to explore. We can help to get iOS and Android dev licenses for you at CfB.

**Comment by JacobEvelyn** [comment]
Meteor plugs in really easily with PhoneGap, just FYI.

**Comment by radhikamalik** [comment]
@WheresHJ still waiting on the iOS license!

**Comment by radhikamalik** [comment]
Closing since duplicate of #135
